apps/discord/app.py

```
@@ -0,0 +1,344 @@
+):
+    """Handle link shortening (legacy function for backward compatibility)."""
+    handler = CommandHandler()
+    options = [{"name": "url", "value": url}]
+    if custom_slug:
+        options.append({"name": "custom_slug", "value": custom_slug})
+
+    await handler.handle_shorten_command(app_id, interaction_token, options)
+
+
+@app.function(secrets=[discord_secret], image=image)
+def test_bot_token():
```

Ruff (0.12.2)

Missing return type annotation for public function `test_bot_token`

Add return type annotation: `bool`

(ANN201)